docs/getstarted.md

```diff
@@ -4,40 +4,33 @@
+For Linux:
 
-1. Download the latest [release](https://github.com/imperialCHEPI/healthgps/releases) *source code* and *binaries* for Linux from the repository.
-2. Unzip both files content into a local directory of your choice (xxx).
-3. Download the [HLM_France](https://github.com/imperialCHEPI/healthgps-examples/tree/main/HLM_France) directory and extract it to a directory named `example` in your Health-GPS directory.
-4. Open a command terminal and navigate to the directory used in step 2 (xxx).
-5. Run: `user@machine:~/xxx$ ./HealthGPS.Console.exe -f healthgps/example/France.Config.json -s healthgps/data` where `-f` gives the *configuration file* full-name and
-`-s` the path to the root folder of the *backend storage* respectively.
-6. The default output folder is `$HOME/healthgps/results/france`, but this can be changed in the *configuration file* `(France.Config.json)`.
+```sh
+./HealthGPS.Console -c https://github.com/imperialCHEPI/healthgps-examples/releases/download/20240907/HLM_France.zip
+```
 
-**NOTE:** *The development datasets provided in this example are limited to 2010-2050 time frame. It is provided for demonstration purpose to showcase the model's usage, input and output data formats. The backend data storage can be populated with new datasets, the [index.json][datastore] file defines the storage structure and file names, it also stores metadata to identify the data sources and respective limits for validation.*
+Output data will be written to a subfolder of your home directory, `healthgps/results/france`.
 
-***Known Issue:*** `Windows 10` support for VT (Virtual Terminal) / ANSI escape sequences is turned OFF by default, this is required to display colours on console / shell terminals. You can enable this feature manually by editing windows [registry keys](https://superuser.com/questions/413073/windows-console-with-ansi-colors-handling/1300251#1300251), however we recommend the use of [Windows Terminal](https://www.microsoft.com/en-gb/p/windows-terminal/9n0dx20hk701?rtc=1&activetab=pivot:overviewtab), which is a modern terminal application for command-line tools, has no such limitation, and is now distributed as part of the `Windows 11` installation.
+## Results
 
-# Results
+NB: For analysis and visualisation of *Health GPS* output data, we recommend the [`healthgpsrvis` R package](https://imperialchepi.github.io/healthgpsrvis/), but here we show how to do some analysis manually.
 
 The current model output format is JSON (JavaScript Object Notation), an open standard file format designed for data interchange in human-readable text. It is language-independent; however, all programming language and major data science tools supports JSON format because they have libraries and functions to read/write JSON structures. To read the model results in [R](https://www.r-project.org/), for example, you need the [`jsonlite`](https://cran.r-project.org/web/packages/jsonlite/vignettes/json-aaquickstart.html) package:
```

Comment:
I'd probably emphasize that the main output is CSV. We haven't ever used the JSON output. and I'm not convinced it is correct. 

Relatedly, the R analysis example is probably inaccurate, and thus misleading..

Reply:
> I'd probably emphasize that the main output is CSV. We haven't ever used the JSON output. and I'm not convinced it is correct.

Sure, will do.

> Relatedly, the R analysis example is probably inaccurate, and thus misleading..

That was my worry... Shall I make an issue for it? Perhaps @SaranjeetKaur could take a look at some point, if she has time. If it *is* inaccurate, we can just delete all that content and signpost the R package instead.

Reply:
Yeah, sounds good. Her stuff is less likely to be wrong, and a shame not to use it.